openmsupply/notify
# 185 configuable temperature limits for cold chain

Closes #185

# 👩🏻‍💻 What does this PR do?
 

![image](https://github.com/openmsupply/notify/assets/8287373/90edb607-bad5-4734-997e-a2d958724f07)

Acceptance criteria
- [ ]  defaults are visible and easy to configure
- [ ]  custom limits can be specified (for all sensors in the config)
- [ ]  Update i icon messages to explain the default limits


# 🧪 How has/should this change been tested?


- Update the limits and check that they are saved correctly.
- Check that limits are disabled when not enabled
- Check defaults are correctly set

## 💌 Any notes for the reviewer?


No backend changes were needed for this, as it was already handled correctly in the backend.
